ocfweb/account/templates/account/print/qr.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+{% load bootstrap %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-8 ocf-content-block">
+            <p>
+                Please bring this QR code to the lab to start your print job. You may want to take a photo of it with your phone.
```

Comment:
Maybe suggest a screenshot (or a very good, even picture)? A typical phone camera picture of a screen containing a QR code might be difficult to scan.

Reply:
We could also automatically emails this QR code to the associated email with the account

Reply:
Ooh, that's smart too!

Reply:
That's a great idea! It seems I need to update ocflib to do that, so I'll do that tonight if I get to it.

Reply:
https://github.com/ocf/ocflib/pull/230

Reply:
Okay I've made this change. It now emails the QR code to you!